source/modules/nphprediction/source/convUtils.py

```diff
@@ -0,0 +1,48 @@
+##                                                                           ##
+##     Copyright (c) 2019                                                    ##
+##     by the Regents of the University of California                        ##
+##                            All rights reserved                            ##
+##                                                                           ##
+## Redistribution and use in source and binary forms, with or without        ##
+## modification, are permitted provided that the following conditions are    ##
+## met:                                                                      ##
+##                                                                           ##
+##     1. Redistributions of source code must retain the above copyright     ##
+##        notice, this list of conditions, and the following disclaimer.     ##
+##                                                                           ##
+##     2. Redistributions in binary form must reproduce the above copyright  ##
+##        notice, this list of conditions, and the following disclaimer in   ##
+##        the documentation and/or other materials provided with the         ##
+##        distribution.                                                      ##
+##                                                                           ##
+##                                                                           ##
+## THIS SOFTWARE IS PROVIDED BY <COPYRIGHT HOLDER> "AS IS" AND ANY           ##
+## EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE         ##
+## IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR        ##
+## PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> OR           ##
+## CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,     ##
+## EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,       ##
+## PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR        ##
+## PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF    ##
+## LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING      ##
+## NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS        ##
+## SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.              ##
+##                                                                           ##
+## The views and conclusions contained in the software and documentation     ##
+## are those of the authors and should not be interpreted as representing    ##
+## official policies, either expressed or implied, of <copyright holder>.    ##
+###############################################################################
+
+def convert_dcm_to_niigz(dir):
+    from nipype.interfaces.dcm2nii import Dcm2niix
+    converter = Dcm2niix()
+    converter.inputs.source_dir = dir
+    converter.inputs.out_filename = '%b'
```

Comment:
Enlighten me on what `%b` does and which `Python` version you are using.

Reply:
So this is for the naming scheme used when the nii file is created - in this case, I’m having it name it the same as the DICOM so we don’t have issues with file names if we want to access/compare for dice and such in the future